code-challenges.test.js

```diff
@@ -44,8 +70,41 @@ const object3 = { number: -7 }
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+describe("randItems", () => {
+    // "randItems" is the string arguement for the describe function
+    it("takes in an array of words and returns an array with all the words capitalized", () => {
+    // the it statement says that the first letter in each element of array randomNouns1 should be capitalized
+        expect(randItems).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    // the expect statements calls the function and uses the .toEqual to compare the output
+    }) 
+}) // FAIL: ReferenceError: randItems is not defined
+describe("randItems2", () => {
+    // "randItems2" is the string arguement for the describe function
+    it("takes in an array of words and returns an array with all the words capitalized", () => {
+    // the it statement says that the first letter in each element of array randomNouns2 should be capitalized
+        expect(randItems2).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+    // the expect statements calls the function and uses the .toEqual to compare the output
+    })
+}) // FAIL: ReferenceError: randItems2 is not defined
```

Comment:
There should be function calls in the expect statement. You have a variable there